OmiseSDK/FPXBankChooserViewController.swift

```diff
@@ -0,0 +1,131 @@
+import UIKit
+import os
+
+
+@objc(OMSFPXBankChooserViewController)
+class FPXBankChooserViewController: AdaptableDynamicTableViewController<Capability.Backend.Bank>, PaymentSourceChooser, PaymentChooserUI {
+    var email: String?
+    var flowSession: PaymentCreatorFlowSession?
+    private let defaultImage: String = "FPX/unknown"
+    private let message = "Cannot retrieve list of banks.\nPlease try again later."
```

Comment:
I would prefer to add all strings to our `Localizable.strings`. Even if we don't want to localize some payment methods now, if we choose to do it later it will be difficult to find the strings in the source code. It's okay when you use the English text for all languages.